xwiki-commons-core/xwiki-commons-xml/src/test/java/org/xwiki/xml/XMLUtilsTest.java

```diff
@@ -65,6 +65,24 @@ void escape()
         assertFalse(escapedText.contains("&&"), "Failed to escape &");
     }
 
+    @Test
+    void escapeMinimal()
+    {
+        String escapedText = XMLUtils.minimalEscape("a < a' && a' < a\" => a < a\"");
+
+        assertFalse(escapedText.contains("<"), "Failed to escape <");
+        assertFalse(escapedText.contains("&&"), "Failed to escape &");
+        assertTrue(escapedText.contains(">"), "Failed to keep >");
+        assertTrue(escapedText.contains("'"), "Failed to keep '");
+        assertTrue(escapedText.contains("\""), "Failed to keep \"");
```

Comment:
I find it safer to assert the exact output and not whether the output contains some strings. It's not like the input string is that long. It's also easier to see the difference between the expected and the actual value when the test fails.

Reply:
I reused the testing style of the test class, but I agree with you. Test updated here https://github.com/xwiki/xwiki-commons/pull/101/files#diff-4d325ecb5d796d402110ecb9c1edfdf7R72